client/src/components/main/moviesList/moviesList.component.js

```diff
@@ -24,3 +25,10 @@ export default function MoviesList(props) {
     </ErrorBoundary>
```

Comment:
There's a good point to wrap ErrorBoundary around the entire app, not just around separate components.
For example, around Header + Main + Footer in app.js.